.ci/Jenkinsfile_cloud

```diff
@@ -2,47 +2,68 @@
 library 'kibana-pipeline-library'
 def loadPipeline
 
-pipeline {
-    agent { label 'docker && ubuntu-tests-l' }
-    stages {
-        stage ('Initialize') {
-            steps {
+timeout(time: 300, units: 'MINUTES') {
+    def workers = [:]
+
+    def numWorkers = params.NUMBER_WORKERS || 3
+    def numTestRuns = params.NUMBER_TEST_RUNS || 30
+    def modTestRuns = numTestRuns % numWorkers
+    def numJobsPerWorker = numTestRuns.intdiv(numWorkers)
+
+    for (def i = 1; i <= 3; i++) {
+        workers["worker-${i}"] = {
+            for (def j = 1; j <= numJobsPerWorker; j++) {
+                runPipeline(i, j)
```

Comment:
@dmlemeshko I'm curious if you have any ideas about how to easily change this so that we could just run the same scenario multiple times, in serial, against the same deployment. This would probably save quite a bit of time if we could just reuse the deployment.

I played around with changing the AtOnceJourney to use `repeat` and changing the numbers around, but neither worked. I'm not very familiar with this, so thought you might have a better idea

Reply:
I'm working on it here #67

The idea is to create deployment and then use it for [comma-separated simulations array](https://github.com/elastic/kibana-load-testing/pull/67/files#diff-316bd874357081f73890574d090af44e8526b502bf6778e29610bfdba5870991R31-R38)
PR assumes we are running different simulations against a single deployment.

For the case of running the same simulation multiple time we can simply add a loop on top of looping simulations in that bash script:

```
for i in ${number_test_runs}
  for j in "${sim_array[@]}"
    ...
```

Reply:
Great, that is fantastic. We can probably merge this first and then I can adapt this to use that option instead.